resources/crates.py

```diff
@@ -0,0 +1,9 @@
+from pulumi_command import local
+
+
+def add_owner(username: str):
+        add_owner_command = local.Command(
+        f"add_owner_{username}",
```

Comment:
This is my first time looking at pulumi.
How do we test these? These commands could have several parameters, and may have typos. What is the workflow for developing these?

Reply:
Good question. I've just been running them locally and validating that they work. This flow is somewhat dangerous, but I'm choosing to do more of an a-b test approach (i.e. only enable for me at first) to limit blast radius.